src/vmm/src/device_manager/mmio.rs

```diff
@@ -486,6 +486,18 @@ impl MMIODeviceManager {
                         // so for Vsock we don't support connection persistence through snapshot.
                         // Any in-flight packets or events are simply lost.
                         // Vsock is restored 'empty'.
+                        // The only reason we still `kick` it is to make guest process
```

Comment:
The commit description should be `fix: ` I think.

Reply:
fixed

Reply:
we would need a changelog entry for this, I guess

Reply:
added